### PR TITLE
Add facetstats for Meilisearch v1.1.0

### PR DESCRIPTION
**Pull request by bidoubiwa**
Add the `facetStats` type in the `SearchResults` structure. It introduces min max values for numeric facet that have been required through the search parameter facet 

As per [the specification](https://github.com/meilisearch/specifications/pull/224)

SDK requirements: https://github.com/meilisearch/integration-guides/issues/251

### CODE REVIEW

File: meilisearch-test-macro/src/lib.rs [hunk 1]
```diff
@@ -169,9 +169,7 @@ pub fn meilisearch_test(params: TokenStream, input: TokenStream) -> TokenStream
 
         outer_fn.sig.inputs.clear();
         outer_fn.sig.asyncness = inner_fn.sig.asyncness.clone();
-        outer_fn
-            .attrs
-            .push(parse_quote!(#[futures_await_test::async_test]));
+        outer_fn.attrs.push(parse_quote!(#[tokio::test]));
```
Comment (alallema) [inline meilisearch-test-macro/src/lib.rs:172]: Oh, what is tokio test?

File: src/tasks.rs [hunk 2]
```diff
@@ -1043,17 +1052,20 @@ mod test {
...
+        mock_res.assert_async().await;
+
         Ok(())
     }
 
     #[meilisearch_test]
     async fn test_delete_tasks_with_params() -> Result<(), Error> {
-        let mock_server_url = &mockito::server_url();
+        let mut s = mockito::Server::new_async().await;
+        //         let mut s = mockito::Server::new_async().await;
```
Comment (alallema) [inline src/tasks.rs:1063]: Shouldn't this line be removed?

**Review by alallema — COMMENTED** [review]
Well done! 🥖

File: src/tasks.rs (hunk 2 above)
Comment (bidoubiwa) [reply src/tasks.rs:1063]: yes!

File: meilisearch-test-macro/src/lib.rs (hunk 1 above)
Comment (bidoubiwa) [reply meilisearch-test-macro/src/lib.rs:172]: Its the library used to handle async testing. We had to change it to be compatible with the new version of Mockito

**Review by alallema — APPROVED** [review]
LGTM! 🎉

**Pull request merged**